message-format.md

```diff
@@ -54,6 +54,8 @@ msg = {
     "seq": 12,
 
     # Announced networks, OPTIONAL
+    # A router with only a IPv4 address MUST NOT advertise IPv6 networks and
+    # vice versa
```

Comment:
why?

Reply:
Can you route an IPv6 Packet to a router that does not have an IPv6 ip address? I thought that is not possible